PKGBUILD

```diff
@@ -511,7 +511,7 @@ DEST_MODULE_LOCATION[3]="/kernel/drivers/video"' dkms.conf
     # 5.11
     if (( $(vercmp "$_kernel" "5.11") >= 0 )); then
       _kernel511="1"
-      _whitelist511=( 460* )
+      _whitelist511=( 460.32* )
```

Comment:
Could you please add 460.27*, which is also affected?

Reply:
I will get on it

Reply:
added